packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -192,9 +192,12 @@ model = GLiNER.from_pretrained("${model.id}")`,
 ];
 
 export const keras = (model: ModelData): string[] => [
-	`from huggingface_hub import from_pretrained_keras
+	`# Available backend options are: "jax", "tensorflow", "torch".
+os.environ["KERAS_BACKEND"] = "tensorflow"
+	
+import keras
 
-model = from_pretrained_keras("${model.id}")
+model = keras.saving.load_model("${model.id}")
```

Comment:
I think a prefix was needed?

```suggestion
model = keras.saving.load_model("hf://${model.id}")
```